### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by embray**
A useful tip to include in the contributing guidelines.

### CODE REVIEW

**Comment by embray** [comment]
Ironically, I did not include "[skip ci]" in the commit message for this change.  Making the edit through the browser threw me off :P

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -46,3 +46,21 @@ include the following:
   or test updates.  Only user-visible changes (new features/API changes, fixed
   issues) need to be mentioned.  If in doubt ask the core maintainer reviewing
   your changes.
+
+Other Tips
+----------
+
+- When contributing trivial documentation fixes (i.e. fixes to typos, spelling,
+  grammar) that do contain any special markup and are not associated with code
```
Comment (mdboom) [inline CONTRIBUTING.md]: Did you mean "that do not contain any special markup"?

Comment (embray) [inline CONTRIBUTING.md]: That I did.

**Comment by mhvk** [comment]
Looks good.

**Comment by eteq** [comment]
Looks good to me, too.  Will go ahead and merge now.

**Pull request merged**